src/Lint.jl

```diff
@@ -350,7 +356,54 @@ function lintdir{T<:AbstractString}(dir::T, ctx::LintContext=LintContext())
     ctx.messages
 end
 
-function readandwritethestream(conn)
+function convertmsgtojson(msgs, style)
+    if style == "LintMessage"
```

Comment:
This style is in CamelCase and the others in snake-case; should be made consistent.